plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_local_config.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass, field
+from typing import Any, Dict
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+
+
+@dataclass
+class RayLocalConf:
+    # ray.init(**ray_init_config)
+    ray_init_cfg: Dict[str, Any] = field(default_factory=dict)
+
+    # @ray.remote(**ray_remote_cfg)
+    ray_remote_cfg: Dict[str, Any] = field(
+        default_factory=lambda: {"num_cpus": 1, "num_gpus": 0}
+    )
+
+
+@dataclass
+class RayLocalLauncherConf(ObjectConf):
+    cls: str = "hydra_plugins.hydra_ray_launcher.ray_local_launcher.RayLocalLauncher"
+    params: RayLocalConf = RayLocalConf()
+
+
```

Comment:
You can use inheritance to reduce duplication with the AWS launcher config (and/or move the data here into a hydra/launcher/ray_local.yaml)